paasta_tools/cli/cmds/rerun.py

```diff
@@ -85,6 +88,21 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    rerun_parser.add_argument(
+        '-t', '--rerun-type',
```

Comment:
I was thinking to have a real parameter in order to make this "policy" configurable.
As I wrote in the PR text I would add in a different PR the possibility to run only the required job and all his childs.